praw/models/redditors.py

```diff
@@ -0,0 +1,49 @@
+"""Provide the Redditors class."""
+from .base import PRAWBase
+from .listing.generator import ListingGenerator
+from .util import stream_generator
+from ..const import API_PATH
+
+
+class Redditors(PRAWBase):
+    """Users is a Listing class that provides various Redditor lists."""
```

Comment:
Replace `Users` with `Redditors`.

Reply:
See #1010 😃

Reply:
Oops, I forgot I made that comment. Thanks for addressing.